include/boost/url/detail/parts_base.hpp

```diff
@@ -11,6 +11,7 @@
 #define BOOST_URL_DETAIL_PARTS_BASE_HPP
 
 #include <boost/url/error.hpp>
+#include <boost/url/string_view.hpp>
```

Comment:
What's this for? The file doesn't use `string_view`

Reply:
`pos_t` is defined in `boost/url/string_view.hpp`

Reply:
maybe that should go in config.hpp? do we have both `pos_t` and `off_t`? We only need one.